tests/contrib/oed/test_linear_models_eig.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+import pytest
+
+import pyro
+import pyro.distributions as dist
+import pyro.optim as optim
+from pyro.infer import Trace_ELBO
+from pyro.contrib.glmm import known_covariance_linear_model
+from pyro.contrib.oed.util import linear_model_ground_truth
+from pyro.contrib.oed.eig import (
+    nmc_eig, posterior_ape, marginal_eig, marginal_likelihood_eig, vnmc_eig, laplace_vi_ape, lfire_eig,
+    donsker_varadhan_eig)
+from pyro.contrib.util import rmv, rvv
+from pyro.contrib.glmm.guides import LinearModelLaplaceGuide
```

Comment:
So I wrote fresh guides for the tests to make them more self-contained & readable. These guides get to be way simpler because they do not have to be prepared to all kinds of models. I could also use the `glmm/guides` versions though if you think it would be better

Reply:
I think what you have here is fine, was just curious